pepys_import/utils/branding_util.py

```diff
@@ -0,0 +1,16 @@
+from pyfiglet import Figlet
```

Comment:
I think we don't have this library in the _requirements.txt_ . Can you add it?

Reply:
Sure, I will add it.